pennylane_lightning_gpu/src/simulator/StateVectorCudaManaged.hpp

```diff
@@ -64,56 +80,40 @@ class StateVectorCudaManaged
     StateVectorCudaManaged(size_t num_qubits)
         : StateVectorCudaBase<Precision, StateVectorCudaManaged<Precision>>(
               num_qubits),
-          gate_cache_(true) {
-        PL_CUSTATEVEC_IS_SUCCESS(custatevecCreate(
-            /* custatevecHandle_t* */ &handle));
-    };
+          gate_cache_(true){};
 
     StateVectorCudaManaged(size_t num_qubits, const DevTag<int> &dev_tag,
                            bool alloc = true)
         : StateVectorCudaBase<Precision, StateVectorCudaManaged<Precision>>(
               num_qubits, dev_tag, alloc),
           gate_cache_(true, dev_tag) {
         BaseType::initSV();
-        PL_CUSTATEVEC_IS_SUCCESS(custatevecCreate(
-            /* custatevecHandle_t* */ &handle));
     };
 
     StateVectorCudaManaged(const CFP_t *gpu_data, size_t length)
         : StateVectorCudaManaged(Util::log2(length)) {
         BaseType::CopyGpuDataToGpuIn(gpu_data, length, false);
-        PL_CUSTATEVEC_IS_SUCCESS(custatevecCreate(
-            /* custatevecHandle_t* */ &handle));
     }
 
     StateVectorCudaManaged(const CFP_t *gpu_data, size_t length,
                            DevTag<int> dev_tag)
         : StateVectorCudaManaged(Util::log2(length), dev_tag) {
         BaseType::CopyGpuDataToGpuIn(gpu_data, length, false);
-        PL_CUSTATEVEC_IS_SUCCESS(custatevecCreate(
-            /* custatevecHandle_t* */ &handle));
     }
 
     StateVectorCudaManaged(const std::complex<Precision> *host_data,
                            size_t length)
         : StateVectorCudaManaged(Util::log2(length)) {
         BaseType::CopyHostDataToGpu(host_data, length, false);
-        PL_CUSTATEVEC_IS_SUCCESS(custatevecCreate(
-            /* custatevecHandle_t* */ &handle));
     }
 
     StateVectorCudaManaged(const StateVectorCudaManaged &other)
         : StateVectorCudaManaged(other.getNumQubits(),
                                  other.getDataBuffer().getDevTag()) {
         BaseType::CopyGpuDataToGpuIn(other);
-        PL_CUSTATEVEC_IS_SUCCESS(custatevecCreate(
-            /* custatevecHandle_t* */ &handle));
     }
 
-    ~StateVectorCudaManaged() {
-        PL_CUSTATEVEC_IS_SUCCESS(custatevecDestroy(
-            /* custatevecHandle_t */ handle));
-    }
+    ~StateVectorCudaManaged() {}
```

Comment:
How about
```suggestion
    ~StateVectorCudaManaged() = default;
```
or just remove the definition?

Reply:
Yep, `default` is fine by me. Thanks @chaeyeunpark